modules/auxiliary/scanner/smb/impacket/dcomexec.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python
+# Copyright (c) 2003-2018 CORE Security Technologies
+#
+# This software is provided under under a slightly modified version
```

Comment:
Hmm... just being pedantic, but if it's [this license](https://github.com/CoreSecurity/impacket/blob/master/LICENSE), it appears to be a modified Apache 1.1 license, which clause 3 might not yet be satisfied...

Also, cool addition!